daisy/compute/compute.go

```diff
@@ -217,7 +217,7 @@ func (c *client) InstanceStopped(project, zone, name string) (bool, error) {
 	switch status {
 	case "PROVISIONING", "RUNNING", "STAGING", "STOPPING":
```

Comment:
What would happen if we return true when status == "STOPPING"? Can we get a performance gain out of that (by unblocking Daisy sooner)?